packages/calypso-build/webpack/sass.js

```diff
@@ -49,14 +54,15 @@ module.exports.loader = ( { includePaths, prelude, postCssOptions, cacheDirector
 		{
 			loader: require.resolve( 'sass-loader' ),
 			options: {
-				prependData: prelude,
+				additionalData: prelude,
```

Comment:
Curious, why did this change?

Reply:
The option has been renamed in `sass-loader` 9.0.0. One of breaking changes there. The reasons are unclear to me. One possible reason is that the option can now both prepend and append stuff -- but that seems to not be true, as it still can only prepend. :wave: @alexander-akait -- why did you decide to do this rename?

Reply:
Two reasons:
- logic unification with `less-loader`/`stylus-loader`, they also have this option
- using `additionalData: (content) => `insertBefore${content}insertAfter` allows to inject own data before/after loader, and if you have complex logic, split it and inject middle (or where you want)

Reply:
Thanks, I understand now. Passing a string can still only prepend, but it's the function argument that can do both :+1: